modules/lightmapper_rd/lm_compute.glsl

```diff
@@ -366,7 +366,9 @@ void main() {
 				);
 
 				for (uint j = 0; j < 4; j++) {
-					sh_accum[j].rgb += light * c[j] * (1.0 / 3.0);
+					// A factor of 8.0 is empirically determined to match non-directional lightmapping
+					// in terms of overall brightness.
```

Comment:
This comment can now be removed since Dario pinpointed where the factor of 8 comes from

Reply:
Sorry if this seems like butting in but I think this is useful in case someone comes back and look around that piece of code in the future.

Reply:
How is it helpful? The comment says that the number is empirically determined (which means it was determined by guessing and checking), which was true at first. But now we have an actual rationale for the value, so the comment shouldn't say that it is a random number arrived at from guessing. Dario's comment explains where the number comes from in a much more complete way, but is way too long to include in a comment

Reply:
> but is way too long to include in a comment

Just note that it is not a random value. optionally add link to a discussion

Reply:
I say leave the comment out for now. I'll be looking at moving these into a common function and adding an actual explanation of what the values do once I find their source.

Reply:
@DarioSamo In this PR or a future one? If the latter, I think the comment should be changed to match state after this PR